### DISCUSSION TITLE
[Symfony FLEX] `tabler.yaml` not copied

### DISCUSSION THREAD

**Discussion by cavasinf** [kevinpapst/TablerBundle#14] [Q&A] [answered]
> I've never configured Symfony FLEX or dev on it.
So I'm only making an issue and not a PR.

When installing the bundle from composer with Symfony FLEX, 
the file `config/packages/tabler.yaml` is not copied from the bundle.

`bundles.php` works fine, but not `tabler.yaml`.

**Comment by kevinpapst** [comment, marked as answer]
Hahaha, yeah the README was copied from the Symfony website. But I did not yet register a Flex recipe.

**Comment by cavasinf** [comment]
Well well well, I remember that someone said to me that this bundle is ready to use 😉

We should use a TODO to remember/write what is missing.
I've already forgot what I've done 3 week ago.

**Comment by kevinpapst** [comment]
I never defined "ready to use". But actually it is working for me exactly as I wanted.
But it is really good that you are helping to shape it, so it is getting better.

I'd say we use the issues as TODO list for now.

**Comment by cavasinf** [comment]
You can maybe add labels to the issues.
Even if all issues are requested/missing functionality. 

We can start using the "best/common practice" with github to track that.

**Comment by kevinpapst** [comment]
Are you willing to join the "TablerBundle" team? Then I would invite you to the repo.

At least for the time until it is stable for both of us.

**Comment by cavasinf** [comment]
Sure, 
will I a have a tshirt with "TablerBundle team" on it 🤔

**Comment by kevinpapst** [comment]
😁 I have added you as contributor, but I don't know what happens now. Did you receive an email?

**Comment by wucherpfennig** [comment]
I will contribute too as soon the bundle matured a bit. Currently I am still implementing lots of functionality in our applications hence I am stuck with adminlte...

**Comment by cavasinf** [comment]
I did receive an email. I was able to see my `Collaborator` badge before accepting it but ok 😄

**Comment by cavasinf** [comment]
@kevinpapst can you delete this issue pls https://github.com/kevinpapst/TablerBundle/issues/11

**Reply by kevinpapst** [reply]
Done. I wonder which permissions you have here. I thought that you could at least delete your own issues. Well, we figure it out over the time 😄
